<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SkeletonVisualizationClass</class>
 <widget class="QMainWindow" name="SkeletonVisualizationClass">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>639</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>SkeletonVisualization</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>751</width>
      <height>601</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
     <item>
      <widget class="QGraphicsView" name="imageView"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="buttonOpen">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>Open image</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonSave">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>Save image</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonRefresh">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>Refresh skeleton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelPruning">
         <property name="text">
          <string>Pruning parameter:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxPruning">
         <property name="maximum">
          <number>100000000</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelArea">
         <property name="text">
          <string>Minimum area:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxArea">
         <property name="maximum">
          <number>100000000</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelTime">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxImage">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Image</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxBones">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Skeleton</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxCircles">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Circles</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxContours">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Bounds</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonQuit">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>Quit</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>771</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionRefresh"/>
    <addaction name="separator"/>
    <addaction name="actionConnect"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuScale">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Scale</string>
    </property>
    <addaction name="actionOn"/>
    <addaction name="actionOff"/>
    <addaction name="separator"/>
    <addaction name="actionOriginal"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionInternal"/>
    <addaction name="actionExternal"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuScale"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Connect Breaks</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Terminate aplication</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionOn">
   <property name="text">
    <string>Zoom On</string>
   </property>
  </action>
  <action name="actionOff">
   <property name="text">
    <string>Zoom Off</string>
   </property>
   <property name="toolTip">
    <string>Zoom Off</string>
   </property>
  </action>
  <action name="actionOriginal">
   <property name="text">
    <string>Original Size</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="actionInternal">
   <property name="text">
    <string>Internal</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionExternal">
   <property name="text">
    <string>External</string>
   </property>
   <property name="toolTip">
    <string>External sceleton</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
